.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -126,7 +126,7 @@ jobs:
             name: DualStack
             clusterName: ${{ parameters.clusterName }}
             ginkgoFocus: '\[Feature:IPv6DualStack\]'
-            ginkgoSkip: 'SCTP|session affinity'
+            ginkgoSkip: 'SCTP|session affinity|should function for service endpoints using hostNetwork' # Cilium dualstack has a known issue with this test https://github.com/cilium/cilium/issues/25135
```

Comment:
Lets make a cilium dualstack template. Making this change impacts other E2E negatively.

Reply:
I created a second one and have this one check to make sure the CNI isn't cilium